templates/docs/examples/patterns/pagination/pagination-verbose.html

```diff
@@ -0,0 +1,10 @@
+{% extends "_layouts/examples.html" %} {% block title %}Pagination / Previous and next buttons only {% endblock %} {% block standalone_css %}patterns_pagination{% endblock %} {% block content %}
+<ol class="p-pagination">
+  <li class="p-pagination__item">
+    <a class="p-pagination__link--previous" href="#previous" title="Previous page"><i class="p-icon--contextual-menu">Previous page</i><span>Previous page</span></a>
```

Comment:
"Previous page" is unnecessarily doubled in the icon.

```suggestion
    <a class="p-pagination__link--previous" href="#previous" title="Previous page"><i class="p-icon--contextual-menu"></i><span>Previous page</span></a>
```

Reply:
Fixed